graphql/authorization/auth.go

```diff
@@ -439,7 +441,7 @@ func (a *AuthMeta) FetchJWKs() error {
 		return err
 	}
 
-	resp, err := http.DefaultClient.Do(req)
+	resp, err := a.httpClient.Do(req)
```

Comment:
use `defer resp.Body.Close()`. In this way same connection would be available for reuse.